### PR TITLE
Debug #96553

### PR DESCRIPTION
**Pull request by RenaudWasTaken**
Creating this PR to try to debug: https://github.com/kubernetes/kubernetes/pull/96553
Please consider this as WIP

/sig node

### CODE REVIEW

**Review by hasheddan — COMMENTED** [review]
Seeing this in the logs:
```
 Running Nvidia installer\n/usr/local/nvidia /\nCreating directory /tmp/extract\nVerifying archive integrity... OK\nUncompressing NVIDIA Accelerated Graphics Driver for Linux-x86_64 418.67..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n
```

Can you try doubling:
https://github.com/kubernetes/kubernetes/pull/96723/files#diff-7629c065680da0396ef2e8d190ce7cdd1dbf2c336f99c22ec543a4be61d74ccdR50

File: test/e2e/scheduling/nvidia-gpus.go [hunk 1]
```diff
@@ -171,6 +171,32 @@ func SetupNVIDIAGPUNode(f *framework.Framework, setupResourceGatherer bool) *fra
...
+			}
+
+			logs, err := e2epod.GetPodLogs(f.ClientSet, f.Namespace.Name, p.Name, p.Spec.InitContainers[0].Name)
+			if err != nil {
+				framework.Logf("Failed to get pod logs with error: %q", err)
+				continue
+			}
+			framework.Logf("Got the following logs from pod %q: %s", p.Spec.InitContainers[0].Name, logs)
+			stdout, stderr, err := f.ExecWithOptions(framework.ExecOptions{
+				Command:       []string{"/bin/bash", "-c", "cat /usr/local/nvidia.nvidia-installer.log"},
```
Comment (ruiwen-zhao) [inline test/e2e/scheduling/nvidia-gpus.go:191]: /usr/local/nvidia/nvidia-installer.log ?

**Comment by SergeyKanzhelev** [comment]
abandoned? PR mentioned to be debuged is merged

/close

**Pull request closed without merging**